### PR TITLE
Update another 11 test modules according to #2776

### PR DESCRIPTION
**Pull request by abalakh**
Part of #2776
Updated following CLI tests:
* contentviews
* docker
* domain
* environment
* fact
* globalparam
* gpgkey
* host
* host_collection
* hostgroup
* lifecycleenvironment

### CODE REVIEW

**Comment by abalakh** [comment]
```
 nosetests tests/foreman/cli/test_contentviews.py
.....S..SS................SS.S.....S...S..S......SS...S.........
----------------------------------------------------------------------
Ran 64 tests in 2778.891s

OK (SKIP=12)

nosetests tests/foreman/cli/test_docker.py
....SS....................SSSSSSSSS..............SSSSSS....................
----------------------------------------------------------------------
Ran 75 tests in 2891.960s

OK (SKIP=17)

nosetests tests/foreman/cli/test_domain.py
.....................................
----------------------------------------------------------------------
Ran 37 tests in 418.358s

OK

nosetests tests/foreman/cli/test_environment.py
............................SS...............
----------------------------------------------------------------------
Ran 45 tests in 503.543s

OK (SKIP=2)

nosetests tests/foreman/cli/test_fact.py                          
....SSSSS
----------------------------------------------------------------------
Ran 9 tests in 11.093s

OK (SKIP=5)

nosetests tests/foreman/cli/test_globalparam.py
...
----------------------------------------------------------------------
Ran 3 tests in 16.575s

OK

nosetests tests/foreman/cli/test_gpgkey.py     
SSS.SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS.............SSSSSSSSSSSSSSSSSSSSSS.S
----------------------------------------------------------------------
Ran 90 tests in 85.982s

OK (SKIP=75)

nosetests tests/foreman/cli/test_host.py
..
----------------------------------------------------------------------
Ran 2 tests in 73.810s

OK

nosetests tests/foreman/cli/test_host_collection.py
..SSSSSS.........................................SSSSSSS.
----------------------------------------------------------------------
Ran 57 tests in 459.613s

OK (SKIP=13)

nosetests tests/foreman/cli/test_hostgroup.py
..................................
----------------------------------------------------------------------
Ran 34 tests in 407.157s

OK

nosetests tests/foreman/cli/test_lifecycleenvironment.py
......................................
----------------------------------------------------------------------
Ran 38 tests in 357.244s

OK
```

File: robottelo/cli/metatest/template_methods.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def test_negative_update(self, data):
     )
     self.assertEqual(new_obj[self.search_key], result[self.search_key])
 
-    # Store the new object for future assertionss and to use its ID
```
Comment (elyezer) [inline robottelo/cli/metatest/template_methods.py]: :+1:

**Comment by elyezer** [comment]
Really good job! +995 −2,060 say it all.

ACK.

**Comment by sthirugn** [comment]
ACK

**Pull request merged**